gemfiles/6.0.gemfile

```diff
@@ -0,0 +1,7 @@
+# This file was generated by Appraisal
```

Comment:
These files should have been generated by running `appraisal` after modifying the `Appraisals` file, but I don't see that here.